### PR TITLE
Dataloader-based evaluation for YOLO models

### PR DESCRIPTION
**Pull request by lzrwch**

### CODE REVIEW

**Comment by ahmed-deeplite** [comment]
Isn’t that already there?

File: deeplite_torch_zoo/src/objectdetection/datasets/coco.py [hunk 1]
```diff
@@ -85,16 +84,19 @@ def __getitem__(self, index):
             conf = torch.tensor([1.0])
             category_id = self._delete_coco_empty_category(category_id)
             category_id = torch.tensor([float(category_id)])
-            label = torch.cat((bbox, category_id, conf))
+            label = torch.cat((bbox, category_id))
```
Comment (ahmed-deeplite) [inline deeplite_torch_zoo/src/objectdetection/datasets/coco.py:87]: during training we don't need the `conf` values?

**Review by ahmed-deeplite — COMMENTED** [review]
Looks good to me, thanks!

**Comment by lzrwch** [comment]
- add mAP comparison between this branch and master for VOC and COCO models
- make sure VOC and COCO training works correctly for a few epochs

**Pull request merged**